app/Http/Livewire/Components/login/PasswordResetComponent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Livewire\Components\Login;
+
+use App\Actions\ForceFillUserPasswordAction;
+use Illuminate\Support\Facades\Log;
+use Livewire\Component;
+
+class PasswordResetComponent extends Component
+{
+
+    public $password;
+    public $password_confirmation;
+
+    protected $rules = [
+        'password'=>'required|between:8,255|confirmed',
+        'password_confirmation' => 'required'
```

Comment:
Maybe I am crazy, but where do you check if the password and password_confirmation fields match each other?